src/snapred/backend/dao/normalization/NormalizationRecord.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from snapred.backend.dao import CrystallographicInfo
```

Comment:
can probably delete lines 5, 6, and 8, doesn't look like you're using those imports